components/SubmissionForm.vue

```diff
@@ -142,37 +148,46 @@
 </template>
 
 <script lang="ts" setup>
-import { ref, watch, type Ref, nextTick } from 'vue'
+import { ref, type Ref, watch, nextTick } from 'vue'
 import * as validations from '../utils/formValidations'
 import { useSubmissionStore } from '~/stores/submissionStore'
 import { useLocaleStore } from '~/stores/localeStore'
 
-enum ValidationReference {
-    Location = 'LOCATION',
-    LastName = 'LAST_NAME',
-    FirstName = 'FIRST_NAME',
-    PrimarySpokenLanguage = 'PRIMARY_SPOKEN_LANGUAGE',
-    SecondarySpokenLanguage = 'SECONDARY_SPOKEN_LANGUAGE'
-}
-
 const submissionStore = useSubmissionStore()
 const localeStore = useLocaleStore()
 
-const validationCheckedPreviously: Ref<boolean> = ref(false)
-
+/*
+These values are used in order to make it more readable but also to add
+interactive for the user. Originally the checks are performed after the
+first submission or the first blur event. Then it checks on change so once
+the user enters a valid format the error goes away
+*/
```

Comment:
What do you think about either rewording or removing this comment?

Reply:
Fine with me. I added it because in an earlier resolved conversation @ermish mentioned this

"Philosophy: Comments should explain "why" and not "what"

let's add a comment here to help the next reader understand why there's a next tick here"

So I thought just so others would understand what is happening they could see in the comment. I can just remove it and add it as a comment to this part of the PR